plugins/lang/mapsmenu.txt

```diff
@@ -217,8 +217,8 @@ CHANGLE_MENU = Verander Map Menu
 [es]
 RESULT_REF = Resultado rechazado
 RESULT_ACC = Resultado aceptado
-VOTE_SUCCESS = Votacion finalizada. Se va a cambiar el mapa a
-VOTE_FAILED = Votacion sin exito
+VOTE_SUCCESS = Votacion finalizada (Tiene: %d) (Necesario: %d) Se va a cambiar el mapa a '%s'.
```

Comment:
Did you pull the translations from google translate ?

(%d/%d) would be better than adding the wrong translation